src/app/listing-view/listing-view.component.ts

```diff
@@ -0,0 +1,27 @@
+import {Component, OnInit} from '@angular/core';
+import {Listing} from '../../model/listing';
+import {ListingService} from '../../services/listing.service';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-listing-view',
+  templateUrl: './listing-view.component.html',
+  styleUrls: ['./listing-view.component.css'],
+})
+export class ListingViewComponent implements OnInit {
+  listings: Listing[];
```

Comment:
I don't think `listings` is necessary. If you're only using it to get `listings1`, you could just do `listings1 = res[4]` and remove the instance variable.

Reply:
good point